sys/include/net/gnrc/tcp/config.h

```diff
@@ -54,22 +79,22 @@ extern "C" {
 /**
```

Comment:
Why is the maximum segment size not configurable? The MSS is usually aligned with the underlying network layer for performance reasons, however is could be customized.

Reply:
I was doubtful about whether MSS can be configured. Thank you for the comment.